client/pages/Login.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Redirect from '../components/redirect';
+
+export default class Login extends React.Component {
+
+  render() {
+    if (this.props.credentials !== null) {
+      return <Redirect to='' />;
+    }
+    return (
+      <div className="login-page">
+        <div>
+          <img src="https://getheavy.com/wp-content/uploads/2019/12/spotify2019-830x350.jpg" alt="" className="logo" />
+        </div>
```

Comment:
Instead of linking to an image on someone else's website, it's better to download the file to your `server/public/images` and link it at `/images/name-of-image.jpg`